packages/schemas/tables/sso_saml_assertion_records.sql

```diff
@@ -0,0 +1,19 @@
+/* init_order = 2 */
+create table sso_saml_assertion_records (
+  tenant_id varchar(21) not null 
+    references tenants (id) on update cascade on delete cascade,
+  /** The globally unique identifier of the assertion record. */
+  id varchar(21) not null,
+  /** The globally unique identifier of the SSO connector. */
```

Comment:
```suggestion
  /** The identifier of the SSO connector. */
```

nit: In this table this can be duplicate